test/verify/check-metrics

```diff
@@ -760,6 +761,45 @@ class TestCurrentMetrics(MachineCase):
         b.wait(lambda: float(b.text("#load-avg .pf-l-flex div:first-child").split()[-1].rstrip(',')) > 15)
         m.execute("systemctl stop load-hog 2>/dev/null || true")  # ok to fail, as the command exits by itself
 
+        # Test podman containers
+        if m.image != "fedora-coreos":
```

Comment:
Please add a `# HACK` comment -- this is just because of the missing busybox image on the fcos image, and offline test VM prep, right? It should generally work on FCOS. Hence, invert the condition and do `return`, so that we don't have to outdent the whole thing later on.

Reply:
Yes, there is no busybox image yet (maybe after someone adds support in cockpit-podman :) )

If we return we miss the user services tests which run on fedora-coreos.

Reply:
Ah right, misread the diff -- then let's just add the hack comment.